libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/maneuver/ManeuverViewBinder.kt

```diff
@@ -3,14 +3,20 @@ package com.mapbox.navigation.dropin.component.maneuver
 import android.transition.Scene
 import android.transition.TransitionManager
 import android.view.ViewGroup
+import com.mapbox.maps.Style
 import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
 import com.mapbox.navigation.core.lifecycle.MapboxNavigationObserver
 import com.mapbox.navigation.dropin.R
 import com.mapbox.navigation.dropin.binder.UIBinder
 import com.mapbox.navigation.dropin.databinding.MapboxManeuverGuidanceLayoutBinding
+import com.mapbox.navigation.dropin.lifecycle.EmptyComponent
+import com.mapbox.navigation.dropin.lifecycle.reloadOnChange
+import kotlinx.coroutines.flow.StateFlow
 
 @OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
-class ManeuverViewBinder : UIBinder {
+class ManeuverViewBinder(
+    private val loadedMapStyle: StateFlow<Style?>
```

Comment:
I wonder how someone will override the `ManeuverViewBinder` 

```
with val maneuver: StateFlow<UIBinder?> get() = _maneuver.asStateFlow()
```

They will also need to have access to the stateflow given by `MapStyleLoader` 🤔 .. Wondering if this means we should expose map styles through `UICoordinator` => `UIComponent`

Reply:
`ManeuverCoordinator` does that. It observes `navigationViewContext.uiBinders.maneuver` and returns custom binder if needed.

Reply:
Resolving this one as addressed, we explored some options and they were not as good as this - or required large changes